nixos/tests/zfs.nix

```diff
@@ -193,6 +196,10 @@ let
 
 in {
 
+  superstable = makeZfsTest "superstable" {
+    zfsSuffix = "SuperStable";
+  };
```

Comment:
Maybe `oldstable` would be better? Because what exactly `superstable` means without context?

Reply:
It has a better "stability" property than the average property (etymology of `super`). I think it's more accurate to describe what it is than `oldstable` given that oldstable and stable doesn't have the same stability quality.

Reply:
I can call it `zfsThatDoesNotEatYourData` if you prefer.

Reply:
> It has a better "stability" property than the average property

But is this something that upstream gives? Otherwise we are just inventing names?

We could call this `oldstable` (because at least it is descriptive of what it does) or something else and add something like this to the `meta.longDescription`:

```
This is the old stable version known to work with LTS kernels and avoids some of the possible data corruption issues that may be introduced in more recent releases.
```

But that is just my 2 cents, I don't use ZFS at all.

Reply:
Well, naming is a privilege of the people putting the work in fixing things. ZFS has proved they are unable to deal with predicted issues, so I think they lost their naming privilege here.

For the long description, I agree.

For old stable, I honestly don't like it again because it doesn't reflect the difference in quality. This can change when ZFS would prove itself again.

Reply:
I don't love `superStable` either. If we're going to be supporting 2.1 long term, would it make more sense to put the versions in the attribute name? e.g. `zfs_2_1` as we do elsewhere?

Reply:
OK, I can do that

Reply:
I'm not changing to `zfsStable` to `zfs_2_1` FWIW BTW. That's the responsibility of the new maintainers of ZFS.

Reply:
(someone has to figure out if 2.2 to 2.1 is possible, etc. I don't want to spend that time personally.)

Reply:
I suppose the naming discussion is now finished, resolving, let me know if you still have concerns.